Cargo.toml

```diff
@@ -160,7 +160,7 @@ members = [
 
 [workspace.package]
 # This version string will be inherited by sui-core, sui-faucet, sui-node, sui-tools, sui-sdk, and sui crates.
-version = "1.5.0"
+version = "1.6.0"
```

Comment:
Spurious change?

Reply:
I think there may generally be some merging issues going on here -- I'm seeing changes from other people's commits in the PR